lib/esbuild.ex

```diff
@@ -203,6 +203,11 @@ defmodule Esbuild do
     {:ok, _} = Application.ensure_all_started(:inets)
     {:ok, _} = Application.ensure_all_started(:ssl)
 
+    if proxy = System.get_env("http_proxy") do
+      %{host: host, port: port} = URI.parse(proxy)
+      :httpc.set_options([{:proxy, {{String.to_charlist(host), port}, []}}])
```

Comment:
I may be wrong but i think this has to be :http?

```suggestion
      :httpc.set_options([{:http, {{String.to_charlist(host), port}, []}}])
```

Reply:
I am wrong, ignore me. :D